egs/thchs30/s5/RESULTS

```diff
@@ -3,35 +3,35 @@ for x in exp/{mono,tri1,tri2b,tri3b,tri4b,tri4b_dnn,tri4b_dnn_mpe}/decode_test_p
 #clean mono,tri1,tri2b,tri3b,GMM,DNN model
 #clean test data
 #phone task
-%WER 31.49 [ 113986 / 362027, 20820 ins, 22043 del, 71123 sub ] exp/mono/decode_test_phone/wer_5
-%WER 20.56 [ 74445 / 362027, 15452 ins, 12457 del, 46536 sub ] exp/tri1/decode_test_phone/wer_5
-%WER 17.32 [ 62689 / 362027, 11937 ins, 11260 del, 39492 sub ] exp/tri2b/decode_test_phone/wer_6
-%WER 15.98 [ 57853 / 362027, 8186 ins, 12404 del, 37263 sub ] exp/tri3b/decode_test_phone/wer_9
-%WER 19.67 [ 71215 / 362027, 10156 ins, 15669 del, 45390 sub ] exp/tri3b/decode_test_phone.si/wer_9
-%WER 14.28 [ 51699 / 362027, 7307 ins, 10875 del, 33517 sub ] exp/tri4b/decode_test_phone/wer_9
-%WER 17.54 [ 63483 / 362027, 9898 ins, 12935 del, 40650 sub ] exp/tri4b/decode_test_phone.si/wer_9
-%WER 10.27 [ 37173 / 362027, 8675 ins, 6483 del, 22015 sub ] exp/tri4b_dnn/decode_test_phone/wer_4
-%WER 10.11 [ 36591 / 362027, 8702 ins, 6255 del, 21634 sub ] exp/tri4b_dnn_mpe/decode_test_phone_it1/wer_4
-%WER 10.03 [ 36321 / 362027, 7490 ins, 6731 del, 22100 sub ] exp/tri4b_dnn_mpe/decode_test_phone_it2/wer_5
-%WER 10.01 [ 36249 / 362027, 7507 ins, 6677 del, 22065 sub ] exp/tri4b_dnn_mpe/decode_test_phone_it3/wer_5
+%WER 34.00 [ 123105 / 362027, 17267 ins, 28402 del, 77436 sub ] exp/mono/decode_test_phone/wer_9_0.0
```

Comment:
Maybe the monophone model building not that stable- I wouldn't worry unless
it persists.

On Fri, Jul 1, 2016 at 12:04 PM, Vijayaditya Peddinti <
notifications@github.com> wrote:

> In egs/thchs30/s5/RESULTS
> <https://github.com/kaldi-asr/kaldi/pull/876#discussion_r69343283>:
>
> > @@ -3,35 +3,35 @@ for x in exp/{mono,tri1,tri2b,tri3b,tri4b,tri4b_dnn,tri4b_dnn_mpe}/decode_test_p
> >  #clean mono,tri1,tri2b,tri3b,GMM,DNN model
> >  #clean test data
> >  #phone task
> > -%WER 31.49 [ 113986 / 362027, 20820 ins, 22043 del, 71123 sub ] exp/mono/decode_test_phone/wer_5
> > -%WER 20.56 [ 74445 / 362027, 15452 ins, 12457 del, 46536 sub ] exp/tri1/decode_test_phone/wer_5
> > -%WER 17.32 [ 62689 / 362027, 11937 ins, 11260 del, 39492 sub ] exp/tri2b/decode_test_phone/wer_6
> > -%WER 15.98 [ 57853 / 362027, 8186 ins, 12404 del, 37263 sub ] exp/tri3b/decode_test_phone/wer_9
> > -%WER 19.67 [ 71215 / 362027, 10156 ins, 15669 del, 45390 sub ] exp/tri3b/decode_test_phone.si/wer_9
> > -%WER 14.28 [ 51699 / 362027, 7307 ins, 10875 del, 33517 sub ] exp/tri4b/decode_test_phone/wer_9
> > -%WER 17.54 [ 63483 / 362027, 9898 ins, 12935 del, 40650 sub ] exp/tri4b/decode_test_phone.si/wer_9
> > -%WER 10.27 [ 37173 / 362027, 8675 ins, 6483 del, 22015 sub ] exp/tri4b_dnn/decode_test_phone/wer_4
> > -%WER 10.11 [ 36591 / 362027, 8702 ins, 6255 del, 21634 sub ] exp/tri4b_dnn_mpe/decode_test_phone_it1/wer_4
> > -%WER 10.03 [ 36321 / 362027, 7490 ins, 6731 del, 22100 sub ] exp/tri4b_dnn_mpe/decode_test_phone_it2/wer_5
> > -%WER 10.01 [ 36249 / 362027, 7507 ins, 6677 del, 22065 sub ] exp/tri4b_dnn_mpe/decode_test_phone_it3/wer_5
> > +%WER 34.00 [ 123105 / 362027, 17267 ins, 28402 del, 77436 sub ] exp/mono/decode_test_phone/wer_9_0.0
>
> Did something go wrong ? There is a 2.5% abs WER increase.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/876/files/e0e0d3a5bdad38a11fac583914476771389c3dff#r69343283>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ADJVuym0m63vzGO55Td3vT0Z5YHqQ9L0ks5qRWTWgaJpZM4JDER6>
> .
>